tests/fileio/fileiotests.fz

```diff
@@ -36,17 +36,22 @@ fileiotests =>
   writer := f.write
   reader := f.read
 
-  say "$dir exists: {(f.exists dir).val}"
+  exists(path string) bool is
```

Comment:
I think we want this in the stdlib.

Reply:
- I'd say separate. The feature then uses the stat effect but that is not really a problem, is it?
- Good points, if the feature returns a bool then it should always return true unless really nothing exists but maybe we should rather not return a bool but a more complex answer that includes your mentioned cases.